src/lxc/conf.c

```diff
@@ -3293,8 +3335,7 @@ int lxc_clear_limits(struct lxc_conf *c, const char *key)
 	bool all = false;
 	const char *k = NULL;
 
-	if (strcmp(key, "lxc.limit") == 0
-	    || strcmp(key, "lxc.prlimit"))
+	if (strcmp(key, "lxc.limit") == 0 || strcmp(key, "lxc.prlimit") == 0)
```

Comment:
Oh, you saw that bug too. :) Funny, I sent a branch this morning that fixes this as well. :)